src/Console/DocBloxErrorChecker.php

```diff
@@ -29,6 +31,16 @@ class DocBloxErrorChecker extends \Asika\SimpleConsole\Console
 
 	protected $helpOptions = ['h', 'help', '?'];
 
+	/** @var App */
+	private $app;
+
+	public function __construct(App $app, array $argv = null)
```

Comment:
Have you tested both this console command?

Reply:
yep, it does work because Dice is creating the console classes and automatically constructs the `App` class :-)

=> Checked it